File: lua/refactoring/tests/refactor/119/rb/custom_extract_var_statement/extract_var.start.rb
def orderCalculation(order, a, b)
  temp = a * b
  blah = (order.quantity * order.itemPrice) - 7

  puts(temp, blah)

  order.quantity * order.itemPrice - max(0, order.quantity - 500) * order.itemPrice * 0.05 + min(order.quantity * order.itemPrice * 0.01, 100)
end

File: lua/refactoring/tests/refactor/119/go/custom_extract_var_statement/extract_var.start.go
package main

import (
    "fmt"
    "math"
)

type Order struct {
    quantity  float64
    itemPrice float64
}

func orderCalculation(order Order, a int, b int) (total float64) {
    temp := a * b
    blah := (order.quantity *
        order.itemPrice) - 7
    fmt.Println(blah, temp)
    return order.quantity*order.itemPrice -
        math.Max(0, order.quantity-500)*order.itemPrice*0.05 +
        math.Min(order.quantity*
            order.itemPrice*0.1, 100)
}
